modules/aws-outposts-load-balancer-clb-fix-subnet.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb-fix-subnet_{context}"]
+= Ensuring AWS Classic Load Balancers use the correct subnet
```

Comment:
Not sure if this was explored, I have only seen CLB. 

The framing is that the issue is due to automatic discovery by the CCM. As I understand, CCM uses CLB. 

So I am not sure if that is a specific problem related to CLBs due to the CCM, or if it can be extrapolated NLBs as well.